torch/_refs/__init__.py

```diff
@@ -2963,6 +2963,7 @@ def _reshape_view_helper(a: TensorLikeType, *shape, allow_copy: bool) -> TensorL
 # NOTE: shape is a vararg because Tensor.reshape can be called with as
 # Tensor.reshape(a, b, c) or Tensor.reshape((a, b, c)) Function call
 # torch.reshape doesn't support unpacked shapes
+@torch.ops.aten.reshape.default.py_impl(DispatchKey.CompositeImplicitAutograd)
 def reshape(a: TensorLikeType, *shape: ShapeType) -> TensorLikeType:
```

Comment:
is this reshape vetted? i saw some discussion on whether we wanted to re-transcribe reshape.